youtube_dl/extractor/xvideos.py

```diff
@@ -76,13 +76,30 @@ class XVideosIE(InfoExtractor):
     }, {
         'url': 'https://de.xvideos.com/video4588838/biker_takes_his_girl',
         'only_matching': True
+    }, {
+        'url': 'https://www.xvideos2.com/video4588838/biker_takes_his_girl',
+        'only_matching': True
+    }, {
+        'url': 'https://www.xvideos3.com/video4588838/biker_takes_his_girl',
+        'only_matching': True
     }]
 
+    HOSTS = ['xvideos', 'xvideos2', 'xvideos3']
+
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(
-            'https://www.xvideos.com/video%s/' % video_id, video_id)
+        self.report_download_webpage(video_id)
+        for i, host in enumerate(self.HOSTS):
+            print(host)
+            webpage = self._download_webpage(
+                'https://www.' + host + '.com/video%s/' % video_id, video_id,
+                note=False, fatal=host == self.HOSTS[-1], errnote=host == self.HOSTS[-1])
+            if webpage:
+                temp_host = self.HOSTS[0]
+                self.HOSTS[0] = host
+                self.HOSTS[i] = temp_host
```

Comment:
Uppercase is used for const.

Reply:
only HOSTS is a const others are all variable.

Reply:
Const means does not change after initial assignment.